weld/ejb/src/main/java/org/jboss/as/weld/ejb/StatefulSessionObjectReferenceImpl.java

```diff
@@ -77,7 +79,8 @@ public StatefulSessionObjectReferenceImpl(EjbDescriptorImpl<?> descriptor) {
     @SuppressWarnings({ "unchecked" })
     public synchronized <S> S getBusinessObject(Class<S> businessInterfaceType) {
         if (isRemoved()) {
-            throw WeldLogger.ROOT_LOGGER.ejbHashBeenRemoved();
+            // Cannot be placed in WeldLogger
+            throw new NoSuchEJBException("EJB has been removed");
```

Comment:
Well, `javax.ejb.NoSuchEJBException` semantics is clearly defined by the EJB spec, that's why I thought it's not worth creating a new logger just for this exception. But I can do this if it's a requirement.